components/das_disconnect_message.py

```diff
@@ -1,29 +1,32 @@
 from components.message import Message
```

Comment:
If you make the change to `__init__.py` above, you'll be able to simplify this to `from components import Message`